docs/command.rst

```diff
@@ -0,0 +1,15 @@
+Command Line
+============
+
+There is an exposed command line interface that is available upon
+installation of the repository.  It can be run with ``orcoursetrion``, and
+``orcoursetrion --help`` will provide the most up to date help information.
+
+The command allows you to run commands that correspond to actions,
+currently the only supported action is ``create_export_repo``, and if
+your environment is setup correctly and you run ``orcoursetrion
```

Comment:
How to know if the env is setup correctly ?  Can you add some info or link to docs ?